lib/internal/test_runner/harness.js

```diff
@@ -186,31 +186,26 @@ async function startSubtest(subtest) {
   await subtest.start();
 }
 
-function test(name, options, fn) {
-  const parent = testResources.get(executionAsyncId()) || getGlobalRoot();
-  const subtest = parent.createSubtest(Test, name, options, fn);
-  return startSubtest(subtest);
-}
-
-function runInParentContext(Factory) {
+function runInParentContext(Factory, addShorthands = true) {
   function run(name, options, fn, overrides) {
     const parent = testResources.get(executionAsyncId()) || getGlobalRoot();
     const subtest = parent.createSubtest(Factory, name, options, fn, overrides);
-    if (parent === getGlobalRoot()) {
-      startSubtest(subtest);
+    if (!(parent instanceof Suite)) {
```

Comment:
if Suite unobservable, or does Suite have a frozen `Symbol.hasInstance`? otherwise someone could break this.

Reply:
`Suite` is not exposed to userland